autoloads/Globals.gd

```diff
@@ -4,6 +4,7 @@ enum InfestationType {NONE, AIR, WATER, GROUND}
 enum ResourceType {NONE, FOOD, FUEL, PARTS, RESEARCH}
 
 var base_infestation_rate = .1
```

Comment:
I'd recommend all caps for things that should be constant, as a convention

Reply:
My thought on base_infestation_rate was that it could be changed based on game settings (example: difficulty), but maybe we could do consts for each difficulty level base infestation rate

Reply:
I would say that's _mostly_ constant then, like the one I added a line below.  Basically saying "you should know what you're doing if you change this"